conf/mccleary.config

```
@@ -0,0 +1,32 @@
+//Profile config names for nf-core/configs
+params {
+    config_profile_description = 'McCleary Cluster at Yale'
+    config_profile_contact = 'Gisela Gabernet'
```

```suggestion
    config_profile_contact = 'Gisela Gabernet'
    config_profile_contact_github = '@ggabernet'
    config_profile_contact_email = '<something>@<something'
```

Need to update the template on this but trying to add these because we've got all kinds of stuff in the contact field 🙈

---

I just added the full contact info, thanks!